test_conformance/subgroups/subhelpers.h

```diff
@@ -31,7 +31,7 @@
 #define NR_OF_ACTIVE_WORK_ITEMS 4
 
 extern MTdata gMTdata;
-typedef std::bitset<128> bs128;
+typedef std::bitset<1280> bs128;
```

Comment:
Looking at the discussion in https://github.com/KhronosGroup/OpenCL-CTS/pull/1621#issuecomment-1429974965 the preferred fix seems to be to limit the local work-group sizes of the various non-uniform subgroup tests.

Reply:
Thanks for pointing out, i will remove that commit.